docs/source/reference/pysupported.rst

```diff
@@ -552,6 +557,122 @@ Finally, here's an example of using a nested `List()`:
    :dedent: 4
    :linenos:
 
+.. _feature-disable-reflected-list:
+
+
+Disabling Reflected Lists
+''''''''''''''''''''''''''
+
+As of Numba 0.47.0 a switch, ``disable_reflected_list``, has been introduced to
+the ``numba.future`` module. Using this switch will disable the
+*reflected-list* fully across the Numba code-base. This means that the built-in
```

Comment:
> fully across the Numba code-base

To me, it sounds like it would affect every module that uses list. 

Maybe rephrase to:

`Referencing ``disable_reflected_list`` in a module will disable *reflected-list* fully in all Numba-compiled code inside that module.`

Reply:
done